sys/include/net/gnrc/rpl.h

```diff
@@ -332,17 +332,19 @@ static inline bool GNRC_RPL_COUNTER_GREATER_THAN(uint8_t A, uint8_t B)
 #define GNRC_RPL_DEFAULT_WAIT_FOR_DAO_ACK (3)
 #endif
 #ifndef GNRC_RPL_REGULAR_DAO_INTERVAL
-#define GNRC_RPL_REGULAR_DAO_INTERVAL (60)
+#define GNRC_RPL_REGULAR_DAO_INTERVAL (60000)
 #endif
 #ifndef GNRC_RPL_DEFAULT_DAO_DELAY
-#define GNRC_RPL_DEFAULT_DAO_DELAY (1)
+#define GNRC_RPL_DEFAULT_DAO_DELAY (1000)
```

Comment:
Could you please add a small description with these defines that includes the unit? Something like the comment below would be perfect to prevent confusion. And maybe rename them to something more consistent such as `GNRC_RPL_DAO_*` (I thing @smlng also suggested something like that).